clients/src/test/java/org/apache/kafka/common/network/EchoServer.java

```diff
@@ -38,6 +38,11 @@
  * A simple server that takes size delimited byte arrays and just echos them back to the sender.
  */
 class EchoServer extends Thread {
+    // JDK has a bug where sockets may get blocked on a read operation if they are closed during a TLS handshake.
```

Comment:
Could you add reference to the comment?

Reply:
Thanks for the review! Addressed in [7ac5fca](https://github.com/apache/kafka/pull/16354/commits/7ac5fca63c6a8325043e2a601e6f712990a74771)